linera-service-graphql-client/gql/service_schema.graphql

```diff
@@ -140,19 +140,19 @@ type ChainStateExtendedView {
 	Hashes of all certified blocks for this sender.
 	This ends with `block_hash` and has length `usize::from(next_block_height)`.
 	"""
-	confirmedLog: LogView_CryptoHash!
+	confirmedLog: _12linera_views02_3a3a8log_view02_3a3a7LogView01_3c12linera_views02_3a3a6common02_3a3a16ContextFromStore01_3c14linera_storage02_3a3a19ChainRuntimeContext01_3c14linera_storage02_3a3a10db_storage02_3a3a9DbStorage01_3c12linera_views02_3a3a6memory02_3a3a11MemoryStore02_2c2014linera_storage02_3a3a10db_storage02_3a3a9WallClock04_3e3e2c2012linera_views02_3a3a6memory02_3a3a11MemoryStore03_3e2c2011linera_base02_3a3a6crypto02_3a3a10CryptoHash01_3e!
 	"""
 	Sender chain and height of all certified blocks known as a receiver (local ordering).
 	"""
-	receivedLog: LogView_ChainAndHeight!
+	receivedLog: _12linera_views02_3a3a8log_view02_3a3a7LogView01_3c12linera_views02_3a3a6common02_3a3a16ContextFromStore01_3c14linera_storage02_3a3a19ChainRuntimeContext01_3c14linera_storage02_3a3a10db_storage02_3a3a9DbStorage01_3c12linera_views02_3a3a6memory02_3a3a11MemoryStore02_2c2014linera_storage02_3a3a10db_storage02_3a3a9WallClock04_3e3e2c2012linera_views02_3a3a6memory02_3a3a11MemoryStore03_3e2c2012linera_chain02_3a3a10data_types02_3a3a14ChainAndHeight01_3e!
 	"""
 	Mailboxes used to receive messages indexed by their origin.
 	"""
-	inboxes: ReentrantCollectionView_Origin_InboxStateView!
+	inboxes: _12linera_views02_3a3a25reentrant_collection_view02_3a3a23ReentrantCollectionView01_3c12linera_views02_3a3a6common02_3a3a16ContextFromStore01_3c14linera_storage02_3a3a19ChainRuntimeContext01_3c14linera_storage02_3a3a10db_storage02_3a3a9DbStorage01_3c12linera_views02_3a3a6memory02_3a3a11MemoryStore02_2c2014linera_storage02_3a3a10db_storage02_3a3a9WallClock04_3e3e2c2012linera_views02_3a3a6memory02_3a3a11MemoryStore03_3e2c2012linera_chain02_3a3a10data_types02_3a3a6Origin02_2c2012linera_chain02_3a3a5inbox02_3a3a14InboxStateView01_3c12linera_views02_3a3a6common02_3a3a16ContextFromStore01_3c14linera_storage02_3a3a19ChainRuntimeContext01_3c14linera_storage02_3a3a10db_storage02_3a3a9DbStorage01_3c12linera_views02_3a3a6memory02_3a3a11MemoryStore02_2c2014linera_storage02_3a3a10db_storage02_3a3a9WallClock04_3e3e2c2012linera_views02_3a3a6memory02_3a3a11MemoryStore03_3e3e3e!
```

Comment:
Mmm ok, this is too much. At this point, how about hashing the typename and use hexadecimal digits? `Type_acbd3412baaaaaaaaabbbb000`.

Reply:
Also possible: a mix of GraphQL type + typename hash: `ReentrantCollectionView_Origin_InboxStateView_Nonnull_12312bdcasd`

Reply:
I think that's much better :)  I've updated this PR.